index.html

```diff
@@ -4,7 +4,7 @@
     <link rel="stylesheet" type="text/css" href="style/style.css">
   </head>
   <body>
-    <script src="dist/ol-debug.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.4.2/ol.js"></script>
```

Comment:
Have you you tried using the unpkg CDN? Something like https://unpkg.com/openlayers@^4/dist/ol.js would always give us the latest OpenLayers 4 version, so we can make sure OLE runs with the latest release.

Reply:
OK good idea, can we be sure that the updates of OL4 won't break in a way our current code? :)
If yes I just redo a new PR

Reply:
I guess OL4 won't change to much and we should stay compatible with the current release. The index.html would be a good initial test case for that.

Reply:
OK :+1:  just made another PR